app/src/main/java/ch/epfl/sdp/artificial_intelligence/GenPoint.java

```diff
@@ -0,0 +1,22 @@
+package ch.epfl.sdp.artificial_intelligence;
+
+public abstract class GenPoint {
+    float arg1;
```

Comment:
I think it is better to use a more meaningful variable name.

Reply:
I named it arg1 and arg2 because it is more general than x,y (cartesian) or φ,θ (polar), what name do you suggest?